tidy3d/plugins/dispersion/fit_web.py

```diff
@@ -296,7 +295,7 @@ def fit(  # pylint:disable=arguments-differ, too-many-locals
 
         # get url
         url_server = self._set_url("default")
-        headers = self._setup_server(url_server)
+        headers = get_headers()
```

Comment:
As for removing the authentication test as quoted below, it's because it still errors, and don't know why.
```
        resp = requests.get(
            f"{url_server}/health/access", headers=headers, verify=Config.ssl_verify
        )
        try:
            resp.raise_for_status()
        except Exception as e:
            raise WebError("Authorization to the server failed. Please try again.") from e
```

Reply:
Ok I just made the change, could you take another quick look and see if I did what you were looking for? thanks